common/scala/src/main/scala/org/apache/openwhisk/core/entity/Exec.scala

```diff
@@ -323,7 +323,10 @@ object Exec extends ArgNormalizer[Exec] with DefaultJsonProtocol {
           // map "default" virtual runtime versions to the currently blessed actual runtime version
           val manifest = execManifests.resolveDefaultRuntime(kind) match {
             case Some(k) => k
-            case None    => throw new DeserializationException(s"kind '$kind' not in $runtimes")
+            case None =>
+              throw new DeserializationException(
+                s"the specified runtime '$kind' is not supported by this platform. Valid values are: ${runtimes
+                  .mkString("'", "', '", "'")}")
```

Comment:
A small nit: How about adding a bracket here?
I think it would be more intuitive.
```
Valid values are: ['nodejs:12', 'java:8', 'python:3']
```

Reply:
I would opt for no brackets. The ‘’ around each and comma separated list I think gets the point across.

Reply:
okay